src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -116,9 +119,39 @@ const useFagsakApi = (
                 ? BehandlingResultat.OPPHØRT
                 : context.behandlingResultat;
 
+        const mapTilPeriodeResultater = (
+            samletVilkårResultat: IVilkårResultat[]
+        ): IRestPeriodeResultat[] => {
+            const identResultater: Map<string, IRestVilkårResultat[]> = new Map();
+            samletVilkårResultat.map(resultat => {
+                if (identResultater.has(resultat.personIdent)) {
+                    // @ts-ignore
```

Comment:
Maybe you can comment on which error was ignored

Reply:
I guess that the linter does not understand what is put in the map. So we need to explicitly type the object.